crates/oxc_linter/src/rules/eslint/sort_imports.rs

```diff
@@ -581,7 +580,7 @@ fn test() {
         (
             "import b from 'b';
 
-            import a from 'a';",
+import a from 'a';",
```

Comment:
Hmm, this code should pass the rule even if this import has spaces in front of it. It does in ESLint: https://eslint.org/play/#eyJ0ZXh0IjoiaW1wb3J0IHsgZm9ybWF0IH0gZnJvbSBcIm94Zm10XCI7XG5cbiAgICAgICAgIGltcG9ydCB7IGRlZmluZVJ1bGUgfSBmcm9tIFwib3hsaW50XCI7XG4iLCJvcHRpb25zIjp7InJ1bGVzIjp7InNvcnQtaW1wb3J0cyI6WyJlcnJvciIseyJhbGxvd1NlcGFyYXRlZEdyb3VwcyI6dHJ1ZX1dfSwibGFuZ3VhZ2VPcHRpb25zIjp7InBhcnNlck9wdGlvbnMiOnsiZWNtYUZlYXR1cmVzIjp7fX0sImVjbWFWZXJzaW9uIjoyMDI2fX19

Reply:
Ah, can we include both still?